### PR TITLE
automatic directory creation when using wenv new with directory

### PR DESCRIPTION
**Pull request by jmpohl**
automatically handle `wenv new dir/wenv` directory creation so you don't have to do it manually

### CODE REVIEW

**Comment by dgrisham** [comment]
Ah, took me a second to realize what this was for but good idea :ok_hand: all of the ones I tested on already existed so I created the directory manually, but this is a good idea.

**Comment by dgrisham** [comment]
Also, branch naming nit: I'd call this something like `fix/create-wenv-parent-dirs` or something (specifically the prefix, in this case could also be `feat` for feature, but thinking `fix` because it's natural to think it would already work this way).

File: wenv [hunk 1]
```diff
@@ -324,6 +324,7 @@ OPTIONS
     [[ -z "$1" ]] && return 1
     local wenv="$1"
 
+    mkdir $WENV_CFG/wenvs/$(dirname $wenv)
```
Comment (dgrisham) [inline wenv:327]: We'll need to use the `-p` flag in case there are nested directories (to create all parent directories as well).

**Comment by jmpohl** [comment]
> Also, branch naming nit: I'd call this something like `fix/create-wenv-parent-dirs` or something (specifically the prefix, in this case could also be `feat` for feature, but thinking `fix` because it's natural to think it would already work this way).

Exactly the kind of feedback I am looking for

**Comment by dgrisham** [comment]
@jmpohl Last step: rebase this on the current master branch (since we merged that other PR). Makes for a clean git history.

**Comment by dgrisham** [comment]
You'll have to force push to this branch after rebasing

**Comment by jmpohl** [comment]
Think I did that right. Let me know.

**Comment by dgrisham** [comment]
@jmpohl yep :ok_hand: the way you can check is switch to the feature branch, then run `git diff master` to see if only the changes you made show up.

**Pull request merged**